apps/ftu/js/wifi.js

```diff
@@ -377,7 +389,11 @@ var WifiUI = {
           }
           // Show connection status
           icon.classList.add('wifi-signal');
-          if (WifiHelper.isConnected(network)) {
+          var connectionStatus = WifiHelper.getNetworkStatus(network) ||
+                                'disconnected';
```

Comment:
nit:

we could make the literal `disconnected` a constant, same for `connected`. But again, not necessary.